sphinx/themes/basic/layout.html

```diff
@@ -103,8 +103,24 @@
 {%- macro css() %}
     <link rel="stylesheet" href="{{ pathto('_static/' + style, 1) }}" type="text/css" />
     <link rel="stylesheet" href="{{ pathto('_static/pygments.css', 1) }}" type="text/css" />
+    {#- Process the custom css files, with their alternate and title properties if available #}
     {%- for cssfile in css_files %}
-    <link rel="stylesheet" href="{{ pathto(cssfile, 1) }}" type="text/css" />
+        {%- if css_props[cssfile] is not defined %}
+            {%- set alt = False %}
+            {%- set csstitle = undefined %}
+        {%- else %}
+            {%- set csstitle = css_props[cssfile].title %}
+            {%- set alt = (csstitle is defined) %}
+            {%- if css_props[cssfile].alternate is defined %}
+                {%- set alt = css_props[cssfile].alternate %}
+            {%- endif %}
+        {%- endif %}
+        {%- if alt %}
+            {%- set rel = "alternate stylesheet" %}
+        {%- else %}
+            {%- set rel = "stylesheet" %}
+        {%- endif %}
+    <link rel="{{ rel }}" href="{{ pathto(cssfile, 1) }}" type="text/css"{% if csstitle is defined %} title="{{ csstitle }}"{% endif %} />
```

Comment:
Agreed, `title=""` is not good. But I also agreed to this is too complicated.

I prefer to pack the CSS information to single object in python side.
For example:
```
{%- for css in css_files %}
<link rel="{{ css.rel }}" href="{{ pathto(css.filename, 1) }}" type="text/css"{% if css.title is none %} title="{{ css.title }}"{% endif %} />
<%- endfor %}
```

How about this?